<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMasterCell</class>
 <widget class="QWidget" name="PageMasterCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Specifications">
      <attribute name="title">
       <string>Specifications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ParamTable" name="specificationsTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Balancing">
      <attribute name="title">
       <string>Balancing</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ParamTable" name="balancingTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Throttling">
      <attribute name="title">
       <string>Throttling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="ParamTable" name="throttlingTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SoC">
      <attribute name="title">
       <string>SoC</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="ParamTable" name="socTab"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamTable</class>
   <extends>QTableWidget</extends>
   <header>widgets/paramtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
